src/pkg/term/colorizer.go

```diff
@@ -158,7 +184,8 @@ func (t *Term) Printlnc(c Color, v ...any) (int, error) {
 }
 
 func (t *Term) Printfc(c Color, format string, v ...any) (int, error) {
-	return outputf(t.stdout, c, format, v...)
+	line := ensureNewline(fmt.Sprintf(format, v...))
```

Comment:
why was this changed?

Reply:
Removed the less usefull `outputf` func above